python/gpufort/util/parsing/parsing.py

```diff
@@ -1942,6 +1942,10 @@ def is_declaration(tokens):
 def is_blank_line(statement):
     return not len(statement.strip())
 
+def is_acc_rtlib(statement, modern_fortran):
```

Comment:
Won't accept frontend changes now as I am currently changing it on a separate development branch.
I suggest to keep a local copy and not include to this PR.

Reply:
I'll remove it in the next PRs